src/ngraph/frontend/onnx_import/op/topk.cpp

```diff
@@ -79,6 +92,31 @@ namespace ngraph
                 }
             }
 
+            namespace set_11
+            {
+                NodeVector topk(const Node& node)
+                {
+                    // Process inputs
+                    auto data = node.get_ng_inputs().at(0);
```

Comment:
```suggestion
                    auto data = get_data(node);
```
Is it worth to add a new function for consistency?

Reply:
I don't think so. If the function did any preparation of the input node, then yes, but in this situation I think this is clearer.